tox.ini

```diff
@@ -15,3 +15,11 @@ commands =
 envlist = py27
 commands = python setup.py build_sphinx --fresh-env
 
+[testenv:pep8]
+deps = -r{toxinidir}/test-requirements.txt
+commands = flake8 {posargs}
+
+[flake8]
+show-source = True
+max-line-length = 230
+exclude = .venv,.git,.tox,dist,doc,*egg,build,netman/api/doc_config,netman/adapters/switches/juniper/__init__.py
```

Comment:
Because flake8 complains that an import is not used in this file but it's necessary for the rest of the code base to work properly.